Simplified/Book/UI/NYPLBookDetailView.m

```diff
@@ -610,9 +610,17 @@ - (void)setState:(NYPLBookState)state
       self.buttonsView.hidden = NO;
       self.buttonsView.state = NYPLBookButtonsStateDownloadInProgress;
       break;
+    case NYPLBookStateDownloadingUsable:
+      self.downloadFailedView.hidden = YES;
+      [self hideDownloadingView:NO];
+      self.buttonsView.hidden = NO;
+      self.buttonsView.state = NYPLBookButtonsStateDownloadingUsable;
+      break;
     case NYPLBookStateDownloadFailed:
+      self.downloadFailedView.showAudiobookError = self.book.defaultBookContentType == NYPLBookContentTypeAudiobook;
```

Comment:
since this view is reused (I'm guessing), I was wondering if we need to reset the `showAudiobookError` flag to `NO` in the other cases of the switch, to avoid for example showing the error after a successful retry.

Reply:
This flag set an audiobook specific error message to the `message label` within the `downloadFailedView`. Even if it is `true` after a successful retry, the `downloadFailedView` should still be hidden.